### ISSUE TITLE
 Pls clarify step 39 js beta - build a music player 

### ISSUE THREAD

**Issue by hbar1st** [freeCodeCamp/freeCodeCamp#55389]
### Describe the Issue

please clarify the instructions for step 39 of JS Beta Build a Music Player 
Currently they consist of 2 sentences:
"Add an else block to handle the current song’s position in the playlist.

Within the else block, set the currentTime property of the audio object to the value stored in userData?.songCurrentTime."

The first sentence doesn't seem to relate to the second. It is not clear how 'the current song's position in the playlist" is affected by the actions requested of setting the audio object's currentTime? This makes me think that there's a mistake in the first sentence (or maybe I've misunderstood what it is trying to say so it needs more clarification)

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-basic-string-and-array-methods-by-building-a-music-player/step-39

### Your code

N/A

### Expected behavior

N/A

### System

N/A

**Comment by lasjorg** [comment]
I don't think it has anything to do with the playlist order. The only thing `songCurrentTime` is used for is pause/play (it gets reset everywhere else).

Maybe it can just be:

> Add an `else` block to handle the song's current playback time.

**Comment by a2937** [comment]
My thoughts was that the else block was designed to continue playing the song at the place you stopped it. The current song has been chosen and it's same id of the song. In the `if` block, we're setting the time to zero. In the else block, we're setting it at the old time.

**Comment by lasjorg** [comment]
@hbar1st In what step did you test it?

If you go to the last step, the play/pause should work as intended (or to whatever step has it implemented).

---

Pausing saves the audio element's current time to `songCurrentTime`, and playing restores it back to the element's current time when `songCurrentTime` is available.

**Comment by hbar1st** [comment]
I see, I thought I would be able to pause and play something else then come back to the original selection and it would start from where I stopped (but I don't think the code was created for that).

**Comment by a2937** [comment]
Nah. It only resumes and plays back for the current song. Once you switch songs, it forgets where you left off on the old one.

**Comment by naomi-lgbt** [comment]
I would be fine adding something like "This allows you to resume the current song at the point where it was paused."

**Comment by lasjorg** [comment]
This looks like something that can be fixed by "first time" code contributors to this repository. Here is the file that you should be looking at to work on a fix:


https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/656472ed8f552d2f2b3f7883.md

Change:

```md
Add an `else` block to handle the current song's position in the playlist. 
```

To:

```md
Add an `else` block to handle the song's current playback time. This allows you to resume the current song at the point where it was paused.
```

---

**Please make sure** you read [our guidelines for contributing](https://contribute.freecodecamp.org/#/), we prioritize contributors following the instructions in our guides. Join us in our chat room or the forum if you need help contributing, our moderators will guide you through this.

Sometimes we may get more than one pull-requests. We typically accept the most quality contribution followed by the one that is made first.

Happy contributing.
